.gitlab-ci.yml

```diff
@@ -671,13 +673,16 @@ publish-draft-release:
 
 publish-to-crates-io:
   stage:                           publish
+  needs:
+    - job:                         unleash-check
   <<:                              *docker-env
-  rules:
-    - if: $CI_COMMIT_REF_NAME =~ /^ci-release-.*$/
-    - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
```

Comment:
We should run `cargo unleash em-dragons` only on the version bumps, please return it.